proof/invariant-abstract/ARM/ArchVSpaceEntries_AI.thy

```diff
@@ -1521,8 +1521,9 @@ lemma handle_invocation_valid_pdpt[wp]:
      (auto simp: ct_in_state_def elim: st_tcb_ex_cap)
 
 
-crunch handle_event, activate_thread, switch_to_thread,
-       switch_to_idle_thread, schedule_choose_new_thread
+crunch
+  handle_event, activate_thread, switch_to_thread, switch_to_idle_thread,
+  schedule_choose_new_thread, arch_prepare_next_domain
```

Comment:
is this better? it'll cause the context block for diffs to show only "crunch"

Reply:
It looks like we don't have any official style guidelines for crunch but this is what I've always done when the constants being crunched are too long to fit on the same line as the crunch command. I don't know how many of them are due to me but there's quite a lot of examples of this style in the proofs.

Thinking through it now, is it useful for the context block in diffs to show the first few constants being crunched, with no indication that there's more beyond a trailing comma?

Reply:
I guess that's fair. It might still be worth it to have something so that you can search for that particular line in a file, but in a pinch the diff markers should tell you where that is. So I guess apart from a thought experiment, I don't have anything better for this one.